pylmod/client.py

```diff
@@ -1,13 +1,14 @@
 """
-Python interface to MIT Learning Module
+Python interface to MIT Learning Modules service
```

Comment:
I would say, "Contains the Client class for pylmod that includes all API classes. This should be used if you need to use more than just one LMod API" or similar